components/StatesList.js

```diff
@@ -20,7 +21,10 @@ function StatesList({ states }) {
                     ? <ul style={{backgroundColor: 'white', padding: 7}}>
                         {visibleStates.map(state => (
                             <li key={state.code}>
-                                <StateCard state={state} />
+                                <StateCard
+                                    state={state}
+                                    onLearnMoreClick={() => { trackEvent({ category: 'Click', action: 'State list', label: state.name}) }}
```

Comment:
This gets passed down to the `Learn More` button in the state list and records a click on a specific state card in the list.